examples/window/window/app.py

```diff
@@ -183,9 +189,17 @@ def startup(self):
                 btn_do_report,
                 btn_change_content,
                 btn_hide,
+                btn_beep,
             ],
             style=Pack(direction=COLUMN),
         )
+        self.scroller = toga.ScrollContainer(
+            horizontal=False,
+            vertical=True,
+            style=Pack(flex=1, padding=10),
+        )
+        self.scroller.content = self.inner_box
+        self.main_box.add(self.scroller)
```

Comment:
The widget layout you've got here is: content -> main_box -> scroller -> inner_box -> *buttons

AFAICT, the main box isn't adding anything. You should be able to add the scroller directly as the main content.

Reply:
ok, removed main_box and renamed scroller to main_scroller, so it becomes obvious that the scroller represents the main content.